app/src/main/res/layout/help_activity.xml

```diff
@@ -0,0 +1,32 @@
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/drawer_layout"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include
+      android:id="@+id/toolbar"
+      layout="@layout/toolbar" />
+
+    <FrameLayout
+      android:id="@+id/container_body"
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      android:layout_weight="1" />
+  </LinearLayout>
+
+  <fragment
+    android:id="@+id/fragment_navigation_drawer"
+    android:name="org.oppia.app.drawer.NavigationDrawerFragment"
```

Comment:
This file is not present currently.

Reply:
removed help_activity.xml due to dependency